pubspec.yaml

```diff
@@ -16,21 +18,22 @@ transformers:
     - web/basics/finding_shadow_dom_elements/index.html
     - web/basics/using_bind_to_create_a_single_template_instance/index.html
     - web/basics/creating_a_one_time_binding/index.html
+    - web/basics/dynamically_adding_a_polymer_element_to_the_dom/index.html
     - web/basics/using_a_computed_property/index.html
 
     - web/control_flow/conditionally_hiding_an_element/index.html
     - web/control_flow/getting_the_iteration_index_when_looping_over_a_collection/index.html
     - web/control_flow/looping_over_a_collection_using_iterative_templates/index.html
     - web/control_flow/using_conditional_templates/index.html
-    - web/control_flow/using_template_repeat_with_a_table_row_or_an_option/index.html
+#    - web/control_flow/using_template_repeat_with_a_table_row_or_an_option/index.html
```

Comment:
Adding some suggestions to this one so this can be added back, although there is no workaround for `<option>` elements (that I know of). Will need to add this to the reflectable transformer list as well.